.gitignore

```diff
@@ -44,6 +44,8 @@ nosetests.xml
 coverage.xml
 *,cover
 .hypothesis/
+_trial_temp/
+telegram/_trial_temp
```

Comment:
We probably shouldn't need the `telegram/_trial_temp` ignore. Running tests from inside the `telegram` directory is probably not very safe (Python typically includes the current working directory in the `PYTHONPATH` which could result in odd spurious test failures if there are modules with the same name as other top-level packages).